crypto/hpke/hpke.c

```diff
@@ -0,0 +1,4241 @@
+/*
+ * Copyright 2019-2022 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+/**
+ * @file
+ * An OpenSSL-based HPKE implementation of RFC9180
+ */
+
+#include <stddef.h>
+#include <stdint.h>
+#include <string.h>
+
+#include <arpa/inet.h>
```

Comment:
What is this include for?

After you integrate the DHKEM you might want to comment the includes all out to figure out which ones are needed now..

Reply:
that's gone noe